### DISCUSSION TITLE
New files are not compiled

### DISCUSSION THREAD

**Discussion by PetarSlavov** [unoplatform/uno#8647] [Q&A]
All new files that are added to the solution are not included in the build and are listed as miscellaneous files. This blank file with intentional error does not prevent the solution from compiling. All old files are properly build.  

![image](https://user-images.githubusercontent.com/89679515/165787171-65a3d759-9041-481e-b141-8f0104b478e1.png)

**Comment by jeromelaban** [comment]
Thanks for the report. Can you show the solution explorer?

**Reply by PetarSlavov** [reply]
It turns out this is was a problem on my side. I was doing some project renaming a couple of days ago and some items in the Shared.projectitems file did not properly rename. This affected only new files in the project so I didn't immediately noticed the problem and relate it with the renaming process.
